src/queries/types.js

```diff
@@ -1,3 +1,5 @@
+import * as _ from 'lodash';
```

Comment:
it is more lightweight to import specific methods from lodash instead of everything here.

```
import _camelCase from 'lodash/camelCase';
```

we are doing this already elsewhere in the codebase.
i can get you a link if you want to read more about that?!

Reply:
done in 5c4968a26c167de3f0343c155960812f53460942